src/explorer/WebAppTreeItem.ts

```diff
@@ -126,3 +127,20 @@ export class WebAppTreeItem extends SiteTreeItem {
         return await fs.readFile(templatePath, 'utf8');
     }
 }
+
+export class InvalidWebAppTreeItem implements IAzureTreeItem {
+    public static contextValue: string = 'file';
+    public readonly contextValue: string = InvalidWebAppTreeItem.contextValue;
+    public readonly commandId: string = 'appService.showFile';
```

Comment:
Remove the commandId property - it was only used in the FileTreeItem

Reply:
Fixed.